Extension/src/Debugger/configurationProvider.ts

```diff
@@ -114,7 +114,7 @@ class CppConfigurationProvider implements vscode.DebugConfigurationProvider {
 
         // Filter out build tasks that don't match the currently selectede debug configuration type.
```

Comment:
`selected` has an extra letter `e`.

Same here, also should check defaultConfig.request is `launch`